wiki/en/en-Installation-for-Linux.md

```diff
@@ -14,18 +14,20 @@ Although we don't provide official packages for every Linux distribution, you mi
 
 [![Packaging status](https://repology.org/badge/vertical-allrepos/jamulus.svg)](https://repology.org/project/jamulus/versions)
 
-There is also list of unofficial [packages for different distributions here](https://github.com/corrados/jamulus/issues/223).
+There is also list of unofficial [packages for different distributions here](https://github.com/jamulussoftware/jamulus/discussions/914).
 
-### Debian and Ubuntu (amd64 .deb only)
+### Debian and Ubuntu
+
+Jamulus (with GUI) is will be [included in Debian Bullseye (testing)](https://packages.debian.org/bullseye/jamulus) thanks to [mirabilos](https://github.com/mirabilos). If you already use Bullseye, just install Jamulus via `sudo apt-get install jamulus`.
 
 If you're on amd64 **Debian**/**Ubuntu**, you may try the compiled .deb packages from GitHub Actions.
 
-1. Download the jamulus_latest_ubuntu_amd64.deb file from the [Jamulus GitHub release](https://github.com/corrados/jamulus/releases/tag/latest) page
+1. Download the jamulus_latest_ubuntu_amd64.deb file from the [Jamulus GitHub release](https://github.com/jamulussoftware/jamulus/releases/tag/latest) page
 1. Update apt: `sudo apt-get update`
-1. Install the package: `sudo dpkg -i /path/to/jamulus_latest_ubuntu_amd64.deb`
+1. Install the package: `sudo dpkg -i /path/to/jamulus_latest_ubuntu_amd64.deb` and fix the dependencies with `sudo apt-get install -f`.
```

Comment:
It is better to do:
`sudo apt install /path/to/jamulus_latest_ubuntu_amd64.deb`
This will automatically install the dependencies without need to do the `apt-get install -f`.

Reply:
Thanks. Can you apply the proposal here? You should have push access.

Reply:
Ok. Just applied this proposal.